app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html class="no-js" lang="en">
 <head>
-  <link rel="shortcut icon" href="assets/icons/favicon.ico" />
+  <%= favicon_link_tag 'icons/favicon.ico' %>
```

Comment:
It looks like the favicon is under `/app/assets/images/icons/favicon.ico`, which would make me think this line should have `images/icons/favicon.ico` instead of `icons/favicon.ico`.  Thoughts?